app/assets/javascripts/pageflow/ui/views/table_cells/icon_table_cell_view.js

```diff
@@ -0,0 +1,21 @@
+pageflow.IconTableCellView = pageflow.TableCellView.extend({
+  className: 'icon_table_cell',
+
+  update: function() {
+    this.icon = this.attributeValue();
+
+    this.removeExistingIcons();
+
+    this.$el.attr('title', this.setTitle);
+    this.$el.addClass(this.icon);
+  },
+
+  removeExistingIcons: function() {
+    var iconClasses = pageflow.config.availableTextTrackKinds.join(' ');
+    this.$el.removeClass(iconClasses);
+  },
+
+  setTitle: function() {
+    return I18n.t('pageflow.config.text_track_kind.' + this.icon);
+  }
+});
```

Comment:
"Leaf views" like this a easy to test. Since this is a highly reusable view, I'd say it's worth it. You can look at the other table cells for examples.